core/genesis.go

```diff
@@ -238,14 +238,18 @@ func SetupGenesisBlock(db ethdb.Database, genesis *Genesis) (*params.ChainConfig
 	if height == nil {
 		return newcfg, stored, fmt.Errorf("missing block number for head header hash")
 	}
+
```

Comment:
Why this changed?

Reply:
good point. it shouldn't. reverting back to `master` version

Reply:
solved